episodes/default.nix

```
@@ -0,0 +1,49 @@
+{
+  nixpkgs ? builtins.getFlake "nixpkgs",
+  system ? builtins.currentSystem,
+  pkgs ? nixpkgs.legacyPackages.${system},
+  lib ? nixpkgs.lib,
```

```suggestion
  nixpkgs ? fetchTarball "channel:nixpkgs-unstable",
  system ? builtins.currentSystem,
  pkgs ? import nixpkgs { config = {}; overlays = []; inherit system; }
  lib ? nixpkgs.lib,
```

Essentially the same thing but without depending on Flakes :)